config/initializers/0_clowder-config.rb

```diff
@@ -19,13 +19,25 @@
   redis_url = "#{config.dig('inMemoryDb', 'hostname')}:#{config.dig('inMemoryDb', 'port')}"
   redis_password = config.dig('inMemoryDb', 'password')
 
+  # Kafka
+  first_kafka_server_config = config.dig('kafka', 'brokers')[0]
```

Comment:
Or maybe, if you don't want to use `dig`:

```suggestion
  first_kafka_server_config = OpenStruct.new(config.dig('kafka', 'brokers')[0])
```

Reply:
this was my suggestion initially but declined in order to keep it simple, as you also agreed on ;)